### DISCUSSION TITLE
Transition for HTML gallery slides

### DISCUSSION THREAD

**Discussion by rowanwernham** [fancyapps/ui#580] [Q&A]
I cannot find in your docs what the config option is to set the transitions between slides.

I have a series of HTML elements showing in my lightbox, they navigate via click just fine with what looks like the default fade transition, and i can drag to swipe between them. But i want to set the transition on click to 'slide'

in my config i've tried:

```
const fancyboxConfig = {
  transitionEffect: "slide",
};
```

because i used that previously in another project, and a few other guesses like `transition: "slide"` which worked for my carousel.

Does it need to be inside a settings group for the HTML plugin?

Is this in the docs? I've searched!!!

**Comment by fancyapps** [comment]
Hi,

See https://fancyapps.com/fancybox/api/options/#component-hierarchy

```js
Fancybox.bind('[data-fancybox="gallery"]', {
  // Transition effect when changing gallery items
  Carousel: {
    transition: "slide",
  },
  // Disable image zoom animation on opening and closing
  Images: {
    zoom: false,
  },
  // Custom CSS transition on opening
  showClass: "f-fadeIn",
});
```

https://jsfiddle.net/b1n0qjwL/
